### DISCUSSION TITLE
New Feature: WiFi Security

### DISCUSSION THREAD

**Discussion by arislan91** [BartoszCichecki/LenovoLegionToolkit#885] [Ideas]
Hello,

Is it possible the "WiFi Security" feature from "Lenovo Vantage" to be added in future releases of this project?

**Comment by BartoszCichecki** [comment]
Nope, that feature won't be added.  It's a gimmick. The recipe for wifi security is simple: don't use unsecured networks and if you need to, use a VPN service.

**Reply by arislan91** [reply]
I see. I thought that there is some kind of an action that looks for flaws, malicious packages, etc. behind the curtains. However, if it does nothing on the backend then it is not needed.
